hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSEditLog.java

```diff
@@ -1792,7 +1792,7 @@ private void checkForGaps(List<EditLogInputStream> streams, long fromTxId,
       EditLogInputStream elis = iter.next();
       if (elis.getFirstTxId() > txId) break;
       long next = elis.getLastTxId();
```

Comment:
The local variable is redundant now, we can just update L1805 to be:
```java
txId = elis.getLastTxId() + 1;
```

Reply:
Thanks @xkrogen for you suggestion.